### PR TITLE
Make time grains translatable

### PR DESCRIPTION
**Pull request by xrmx**

### CODE REVIEW

File: caravel/models.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 from flask_appbuilder import Model
 from flask_appbuilder.models.mixins import AuditMixin
 from flask_appbuilder.models.decorators import renders
-from flask_babelpkg import gettext as _
+from flask_babelpkg import lazy_gettext as _
```
Comment (mistercrunch) [inline caravel/models.py]: Wait that means you made everything else that didn't need to be lazy use lazy_gettext

Comment (xrmx) [inline caravel/models.py]: The other two users which are exceptions should be fine.

**Comment by mistercrunch** [comment]
Ooops, I can't merge, can you rebase?

**Comment by xrmx** [comment]
Done

**Pull request merged**